proto/lbm/collection/v1/collection.proto

```diff
@@ -0,0 +1,186 @@
+syntax = "proto3";
+package lbm.collection.v1;
+
+option go_package                      = "github.com/line/lbm-sdk/x/collection";
+option (gogoproto.equal_all)           = false;
+option (gogoproto.goproto_getters_all) = false;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
+// Params defines the parameters for the collection module.
+message Params {}
+
+// Contract defines the information of the contract for the collection.
+message Contract {
+  // contract_id defines the unique identifier of the contract.
+  string contract_id = 1;
+  // name defines the human-readable name of the contract.
+  string name = 2;
+  // meta is a brief description of the contract.
+  string meta = 3;
+  // base img uri is an uri for the contract image stored off chain.
+  string base_img_uri = 4;
+}
+
+// FTClass defines the class of fungible token.
+//
+// Since: 0.46.0 (finschia)
+message FTClass {
+  option (gogoproto.goproto_getters)         = true;
+  option (cosmos_proto.implements_interface) = "TokenClass";
+
+  // id defines the unique identifier of the token class.
+  // Note: size of the class id is 8 in length.
+  // Note: token id of the fungible token would be `id` + `00000000`.
+  string id = 1;
+  // name defines the human-readable name of the token class.
+  string name = 2;
+  // meta is a brief description of the token class.
+  string meta = 3;
+  // decimals is the number of decimals which one must divide the amount by to get its user representation.
+  int32 decimals = 4;
+  // mintable represents whether the token class is allowed to mint or burn its tokens.
+  bool mintable = 5;
+}
+
+// NFTClass defines the class of non-fungible token.
+//
+// Since: 0.46.0 (finschia)
+message NFTClass {
```

Comment:
Does this need `contract_id`?

Reply:
Since there are no plans to update x/collection after this PR (of course, except for the bug fixes), it would be a matter of choice. So to minimize the change, I decided to not modify it.